tap_anthem/client.py

```diff
@@ -9,81 +9,18 @@
 from singer_sdk.helpers.jsonpath import extract_jsonpath
 from singer_sdk.streams import RESTStream
 
-
 SCHEMAS_DIR = Path(__file__).parent / Path("./schemas")
 
 
-class anthemStream(RESTStream):
-    """anthem stream class."""
-
-    # TODO: Set the API's base URL here:
-    url_base = "https://api.mysample.com"
-
-    # OR use a dynamic url_base:
-    # @property
-    # def url_base(self) -> str:
-    #     """Return the API URL root, configurable via tap settings."""
-    #     return self.config["api_url"]
-
-    records_jsonpath = "$[*]"  # Or override `parse_response`.
-    next_page_token_jsonpath = "$.next_page"  # Or override `get_next_page_token`.
-
-    @property
-    def http_headers(self) -> dict:
-        """Return the http headers needed."""
-        headers = {}
-        if "user_agent" in self.config:
-            headers["User-Agent"] = self.config.get("user_agent")
-        # If not using an authenticator, you may also provide inline auth headers:
-        # headers["Private-Token"] = self.config.get("auth_token")
-        return headers
```

Comment:
If this tap were to be published on [Meltano hub](https://hub.meltano.com/), it might be useful to add in a default `User-Agent` to indicate to the data provider (Anthem, in this case) that this is an automated extractor running.

Since I'll be running this a few dozen times locally and not on a recurring basis, I didn't think this was important to keep.